inst/htmlwidgets/plugins/sortable-rstudio/rank_list.scss

```diff
@@ -8,7 +8,7 @@
     border: 1px solid $border-color;
     padding: 10px;
     margin: 5px;
-    display: flex;
+    // display: flex;
```

Comment:
if we keep this one, side by side boxes will wrap to the next row on a skinny window, I believe

Reply:
Thanks.